addons/common/fnc_keyPressedQTE.sqf

```diff
@@ -0,0 +1,60 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_keyPressedQTE
+
+Description:
+    Process Quick-Time Key Press
+
+Parameters:
+    _eventQTE - Character to test against Quick-Time Event <STRING>
+
+Example:
+    ["↑"] call CBA_fnc_keyPressedQTE;
+
+Returns:
+    <BOOLEAN> - True if QTE is running
```

Comment:
```suggestion
    True if QTE is running <BOOLEAN>
```
As seen in https://github.com/CBATeam/CBA_A3/blob/master/addons/arrays/fnc_standardDeviation.sqf and others.